wallet-core/newblockchain.md

```diff
@@ -95,6 +95,7 @@ The C interface, any Protobuf models, and integration tests should be third comm
 The above steps are summarized below as a checklist:
 
 * [ ] Add the coin definition to `coins.json` and `TWCoinType`.
+  * [ ] Create tests in `tests/Blockchain/TWCoinTypeTests.cpp`.
```

Comment:
since tests created automatically with codegen, we might not need this?

Reply:
If both code and its tests are generated with every build, there is not much added value, we could just as well remove them :)  I think it is better to have the tests checked in normally, and any small changes to coins.json need to be done explicitly in the test as well.  For now coins, the test can be added manually, or there the generator can help.